src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -2724,6 +2724,14 @@ internal Conversion ClassifyConversionForCast(int position, ExpressionSyntax exp
         /// <returns>The symbol that was declared.</returns>
         public abstract ISymbol GetDeclaredSymbol(LocalFunctionStatementSyntax declarationSyntax, CancellationToken cancellationToken = default(CancellationToken));
 
+        /// <summary>
+        /// Given a compilation unit syntax, get the corresponding Simple Program entry point symbol.
+        /// </summary>
+        /// <param name="declarationSyntax">The syntax node that declares a member.</param>
```

Comment:
>declares a member [](start = 65, length = 17)

Is this correct?

Reply:
> Is this correct?

Yes, a compilation unit with top level statements declares the method. 

---
In reply to: [403267838](https://github.com/dotnet/roslyn/pull/43038#discussion_r403267838) [](ancestors = 403267838)

Reply:
The comment seems confusing since it's not clear what member is being referred to. Consider changing to "The compilation unit that declares the entry point member".

---
In reply to: [403301060](https://github.com/dotnet/roslyn/pull/43038#discussion_r403301060) [](ancestors = 403301060,403267838)